Block/Algolia.php

```diff
@@ -62,6 +63,7 @@ public function __construct(
         $this->httpContext = $httpContext;
         $this->coreHelper = $coreHelper;
         $this->categoryHelper = $categoryHelper;
+        $this->cart = $cart;
```

Comment:
Using "CartHelper" as alias and "$this->cartHelper" as class property may be clearer ?